redwood-treehouse-compose/src/jsMain/kotlin/app/cash/redwood/treehouse/composition.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.cash.redwood.treehouse
+
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import app.cash.redwood.compose.RedwoodComposition
+import kotlinx.coroutines.flow.Flow
+
+// Inline at callsite once https://github.com/Kotlin/kotlinx.serialization/issues/1454 is fixed.
+public fun RedwoodComposition.bind(
```

Comment:
Pulled this into a separate module that applies the redwood plugin. It might be worth annotating this is an internal API annotation given it's not a public API.

Reply:
Hopefully fixed by https://github.com/JetBrains/androidx/pull/371?

Reply:
I think that fix might already be included in the `1.4.0-rc3` compiler release.